src/app/shared.scss

```diff
@@ -111,3 +111,33 @@ td.mat-mdc-cell ul {
     font-size: 1.25rem;
   }
 }
+
+.search-bar-div {
```

Comment:
Counterintuitive but I suggest removing _all_ styling here.  This looks good but it's styled differently from the other search boxes on the site; the default formatting looks OK.

Also here again #204 needs to land before finalizing the positioning of the search box I think.

Reply:
I have removed the styling that makes the search bar look different, so let me know if that looks better to you now. I've aligned the returned projects text with the search bar above the returned data.

Reply:
I pulled out a few more styles that were not using any more.